src/app/systelab-components/datepicker/datepicker.component.ts

```diff
@@ -29,6 +29,7 @@ export class Datepicker implements OnInit, AfterViewInit, DoCheck, OnDestroy {
 	@Input() public inputExpandHeight: boolean;
 	@Input() public markPreviousAfterDate = false;
 	@Input() public inputFontSize: number;
+	@Input() public showTodayButton = true;
```

Comment:
Set showTodayButton to false in order to have the same behaviour than before

Reply:
Done